src/hotspot/share/gc/z/zForwarding.inline.hpp

```diff
@@ -136,8 +136,12 @@ inline uintptr_t ZForwarding::insert(uintptr_t from_index, uintptr_t to_offset,
   const ZForwardingEntry new_entry(from_index, to_offset);
   const ZForwardingEntry old_entry; // Empty
 
+  // Make sure that object copy is finished
+  // before forwarding table installation
+  OrderAccess::release();
```

Comment:
Can we replace it by `OrderAccess::storestore()`? I think LoadStore constraint in `OrderAccess::release()` is not required here.

Reply:
@albertnetymk Thanks for your suggestion.
> Can we replace it by `OrderAccess::storestore()`?

`OrderAccess::release()` has the same implementation as `OrderAccess::storestore()` on AArch64. Therefore, replacing `OrderAccess::release()` does not yield perf improvement.
I found an interesting pull request related to StoreStore: https://github.com/openjdk/jdk/pull/427 (not integrated yet). I am not aware that this patch can make any improvement.

> Furthermore, is it possible to relax `Atomic::load_acquire` in `ZForwarding::at`?

My initial experiment suggests >5% concurrent mark time reduction using `Atomic::load` instead.
I am still working on checking the correctness of the relaxation.